src/main/java/com/kodilla/ecommercee/domain/Product.java

```diff
@@ -1,7 +1,17 @@
 package com.kodilla.ecommercee.domain;
 
+import javax.persistence.Entity;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.ManyToOne;
+
+@Entity
 public class Product {
+    @Id
     private Long id;
     private String productName;
     private String productType;
```

Comment:
To chyba powinny być kolumny??

Reply:
Do implementacji w zadaniu dot. implementacji encji Product. Ja dodałem tylko niezbędne adnotacje, tak żeby aplikacja się nie wysypała.

Reply:
Nie mam więcej pytań :)